tests/test_validity.py

```diff
@@ -44,8 +45,8 @@ def disk_revolve(n, s):
         return (None,
                 {StorageType.RAM: 0, StorageType.DISK: 0}, 0)
     else:
-        revolver = DiskRevolve(n, s, n - s)
-        return (revolver,
+        cp_schedule = DiskRevolve(n, s, n - s)
+        return (cp_schedule,
                 {StorageType.RAM: s, StorageType.DISK: n - s}, 1)
```

Comment:
@Ig-dolci The disk storage limit looks incorrect.

Reply:
Yes. You are sure. In `PeriodicDiskRevolve` and `DiskRevolve`, it is not correct to add limits for disk storage. I am going to fix that.

Reply:
Since we know there are $n$ steps I think we can work out the expected usage.